### PR TITLE
add history recording and overview

### PR DESCRIPTION
**Pull request by tametsi**
- Add history recording
- Add history page to view & load the last runs

### CODE REVIEW

File: src/lib/runner.ts [hunk 1]
```diff
@@ -2,14 +2,16 @@ import type Run from '@lib/run';
...
+	private _rounds: RoundManager;
 
 	constructor(
 		private _run: Run,
 		private _id: string,
 		private _name: string,
 		private _alias?: string
-	) {}
+	) {
+		this._rounds = new RoundManager();
```
Comment (xXjojaXx) [inline src/lib/runner.ts]: Switching from inline- to constructor-initialization is not required and should be undone

File: src/lib/runner.ts [hunk 2]
```diff
@@ -48,13 +50,32 @@ export default class Runner {
...
+			alias: this.alias,
+			rounds: this.rounds.toJSON(),
+		};
+	}
 }
 
 class RoundManager {
 	private _rounds: number[] = [];
 
+	constructor() {}
```
Comment (xXjojaXx) [inline src/lib/runner.ts]: Unused constructor should be removed again

Comment (tametsi) [reply src/lib/runner.ts]: resolved: https://github.com/tametsi/ligas/pull/1/commits/ec83176d8b098346637230f754fca10aa994dab4

File: src/lib/runner.ts (hunk 1 above)
Comment (tametsi) [reply src/lib/runner.ts]: resolved: https://github.com/tametsi/ligas/pull/1/commits/ec83176d8b098346637230f754fca10aa994dab4

**Review by xXjojaXx — APPROVED** [review]
Completed

**Pull request merged**